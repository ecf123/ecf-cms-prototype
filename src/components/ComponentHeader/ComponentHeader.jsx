import React from 'react'
import "./ComponentHeader.scss";

const ComponentHeader = ({title}) => {
  return (
    <div className='component-header'>
        <h1 className='component-header__title'>{title}</h1>
    </div>
  )
}

export default ComponentHeader