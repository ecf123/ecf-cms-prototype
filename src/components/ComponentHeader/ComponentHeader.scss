@use "../../styles/variables" as *;

.component-header{
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 0 40px;
    width: 100%;
    height: 6.7rem;

    &__title{
        font-family: $font-secondary;
        font-weight: $font-weight-regular;
        font-size: $font-size-extra-large;
        color: $color-blue-navy-dark;
    }
}