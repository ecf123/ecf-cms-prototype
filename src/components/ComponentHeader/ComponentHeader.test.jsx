import { render, screen} from "@testing-library/react";
import { customRender } from "../../utils/testUtils";
import ComponentHeader from "./ComponentHeader";

it("should have a title displayed", () => {
    render(<ComponentHeader title="test title"/>);
    const title = screen.getByText("test title");
    expect(title).toBeInTheDocument();
})

it("Should render layout with children", () => {
  const { container } = customRender(
    <ComponentHeader title="test title"/>
  );
  expect(container).toMatchSnapshot();
});