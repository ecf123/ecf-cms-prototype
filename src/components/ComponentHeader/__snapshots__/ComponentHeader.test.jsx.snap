// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render layout with children 1`] = `
<div>
  <div
    class="component-header"
  >
    <h1
      class="component-header__title"
    >
      test title
    </h1>
  </div>
</div>
`;
